site/class-1-play/dylanvanzanten.svg

```diff
@@ -0,0 +1,57 @@
+<svg xmlns="http://www.w3.org/2000/svg">
```

Comment:
Add a `viewBox` on the `<svg>` element.

Also add `font-family`, `font-size`, and `opacity` on any element.